examples/generate/Cargo.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Nicholas Rodrigues Lordello <nlordell@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-ethcontract = { path = "../.." }
+ethcontract = { path = "../..", no-default-features = true }
```

Comment:
:metal:

Reply:
I am interpreting this as a sign of approval :) .

Reply:
It was a great catch!